### ISSUE TITLE
A11y: [Start chat] Input field in 'Join room' lacks programmatic labe...

### ISSUE THREAD

**Issue by mxandreas** [element-hq/element-x-ios#5256]
- **Checkpoint ID:** 11.1.3.1d/5.1
- **View/Component:** Start chat

### Description
Input field in 'Join room' lacks programmatic label.

### Recommendation
Associate visible label with text field via accessibilityLabel.

--- 
*Report Date: 20.02.2026*
